lib/storage.dart

```diff
@@ -387,104 +117,37 @@ Future<void> insertSession(int sessionId, String name) async {
         'UNIQUE(name)'
         ');');
 
-    // Create sessions table
-    batch.execute('CREATE TABLE IF NOT EXISTS sessions ('
-      'id INTEGER PRIMARY KEY AUTOINCREMENT,'
-      'name TEXT NOT NULL'
-      ');');
-
-    // Create cues table
-    batch.execute('CREATE TABLE IF NOT EXISTS cues ('
-      'id INTEGER PRIMARY KEY AUTOINCREMENT,'
-      'session_id INTEGER NOT NULL,'
-      'time_sec INTEGER NOT NULL,'
-      'message TEXT NOT NULL,'
-      'FOREIGN KEY(session_id) REFERENCES sessions(id) ON DELETE CASCADE'
-      ');');
-
-    // Create daily_reset table
-    batch.execute('CREATE TABLE IF NOT EXISTS $_dailyResetTable ('
-        'id INTEGER PRIMARY KEY NOT NULL,'
-        'date INT NOT NULL,'
-        'activity_1_id INTEGER NOT NULL,'
-        'activity_2_id INTEGER NOT NULL,'
-        'activity_3_id INTEGER NOT NULL,'
-        'activity_completed INTEGER NOT NULL DEFAULT 0,'
-        'FOREIGN KEY(activity_1_id) REFERENCES $_activityTable(id),'
-        'FOREIGN KEY(activity_2_id) REFERENCES $_activityTable(id),'
-        'FOREIGN KEY(activity_3_id) REFERENCES $_activityTable(id)'
+    // Create mood journal table
+    batch.execute('CREATE TABLE $_moodJournalsTable ('
+        'id INTEGER PRIMARY KEY AUTOINCREMENT,'
+        'date TEXT NOT NULL,'
+        'mood TEXT NOT NULL,'
+        'intensity INTEGER NOT NULL,'
+        'note TEXT'
```

Comment:
Update the design document with the new ERD table(s) and design rationale for it.